front_camera/front_camera/get_correction_angle.py

```diff
@@ -1,20 +1,23 @@
 """
-指定画像中の背景（の黄色矩形）が中心に来るような補正角度を返す
-@author: bizyutyu YKhm20020 keiya121
+指定画像中の補正対象の色の矩形が中心に来るような補正角度を返す
+@author: bizyutyu YKhm20020 keiya121 CHIHAYATAKU
 """
 
 import argparse
 import os
 import cv2
 import math
 from camera_interface import CameraInterface
-from yellow_rectangle_detector import YellowRectangleDetector
+from color_rectangle_detector import ColorRectangleDetector
 
 class GetCorrectionAngle:
-    def __init__(self):
-        """コンストラクタ"""
+    def __init__(self, color):
+        """コンストラクタ
+        Args:
+            color (str): 補正対象の色
+        """
 
-        self.yellow_rectangle_detector = YellowRectangleDetector()
+        self.rectangle_detector = ColorRectangleDetector(color)
```

Comment:
ColorRectamgleDetectorの例外処理
```suggestion
        try:
            self.rectangle_detector = ColorRectangleDetector(color)
        except ValueError as e:
            print(e)
```